Source/JavaScriptCore/jit/AssemblyHelpers.h

```diff
@@ -1990,12 +1986,13 @@ class AssemblyHelpers : public MacroAssembler {
         storePtr(TrustedImmPtr(nullptr), Address(resultGPR, JSObject::butterflyOffset()));
     }
 
-    JumpList branchIfValue(VM&, JSValueRegs, GPRReg scratch, GPRReg scratchIfShouldCheckMasqueradesAsUndefined, FPRReg, FPRReg, bool shouldCheckMasqueradesAsUndefined, std::variant<JSGlobalObject*, GPRReg>, bool negateResult);
-    JumpList branchIfTruthy(VM& vm, JSValueRegs value, GPRReg scratch, GPRReg scratchIfShouldCheckMasqueradesAsUndefined, FPRReg scratchFPR0, FPRReg scratchFPR1, bool shouldCheckMasqueradesAsUndefined, std::variant<JSGlobalObject*, GPRReg> globalObject)
+    struct BaselineJITGlobalObject { };
```

Comment:
I guess this could be `std::monostate` but that's probably less readable. Can we make this `enum LazyGlobalObjectLoadTag { LazyBaselineGlobalObject }` instead and we can add a new tag entry when we have unlinked DFG?

Reply:
Done.